quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/configuration/mappers/HostnamePropertyMappers.java

```diff
@@ -1,6 +1,14 @@
 package org.keycloak.quarkus.runtime.configuration.mappers;
 
 
+import com.google.common.net.InternetDomainName;
```

Comment:
I would avoid using Guava directly. Although it is available at runtime, we should avoid if possible a compile-time dependency.

In the worst case, you should try using something from Apache libs as we are using it anyways.